packages/react-vapor/src/components/filterBox/FilterBoxSelectors.ts

```diff
@@ -1,12 +1,22 @@
 import * as _ from 'underscore';
+
 import {IReactVaporState} from '../../ReactVapor';
-import {defaultMatchFilter, MatchFilter} from '../../utils/FilterUtils';
+import {FilterUtils} from '../../utils/FilterUtils';
+import {IItemBoxProps} from '../itemBox/ItemBox';
 import {IFilterState} from './FilterBoxReducers';
 
 export interface GetFilterTextProps {
     id: string;
 }
 
+export type MatchFilter = (filterValue: string, item: IItemBoxProps) => boolean;
+
+export const defaultListBoxMatchFilter = (filterValue: string, item: IItemBoxProps) => {

```

Comment:
Why did you move it in the selector? It make more sense in the Utils, no?

Reply:
Because its bound to `IItemBoxProps`, so it belongs somewhere in the `ItemBox` related files. I did not want it to be mistaken for the defaultMatchFilter of the table, which doesn't use `IItemBoxProps`. 
The selectors might not be the right file tho.

Reply:
I think we can scope these filter with something like 
```
FilterUtils. defaultMatchFilter
TableUtils.defaultMatchFilter
```
what do you think ?

Reply:
I don't mind the name, but the location of the code seems really unusual to me

Reply:
FilterUtils are also relevant for the table

Reply:
Moved the defaultListBoxFilter to a new FilterBoxUtils file